.github/workflows/publish.yml

```
@@ -39,7 +39,7 @@ jobs:
         id: auth
         uses: google-github-actions/auth@v0.4.0
         with:
-          credentials_json: ${{ secrets.GCP_CREDENTIALS }}
+          credentials_json: ${{ secrets.gcp_credentials }}
 
       - name: Get Luarocks API Key
         uses: google-github-actions/get-secretmanager-secrets@v0.2.2
```

aqui você precisa declarar um id pra usar o secret depois:
```suggestion
        uses: google-github-actions/get-secretmanager-secrets@v0.2.2
        id: luarocks-credentials
```


pra usar o secret: 
`${{ steps.luarocks-credentials.outputs.LUAROCKS_API_KEY }}`